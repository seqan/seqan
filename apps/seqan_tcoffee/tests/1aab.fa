>hmgl_trybr
KKDSNAPKRAMTSFMFFSSDFRSKHSDLSIVEMSKAAGAAWKELGPEERKVYEEMAEKDKERYKREM
>hmgt_mouse
KPKRPRSAYNIYVSESFQEAKDDSAQGKLKLVNEAWKNLSPEEKQAYIQLAKDDRIRYDNEMKSWEEQMA
E
>hmgb_chite
ADKPKRPLSAYMLWLNSARESIKRENPDFKVTEVAKKGGELWRGLKDKSEWEAKAATAKQNYIRALQEYE
RNGG
>hmgl_wheat
DPNKPKRAPSAFFVFMGEFREEFKQKNPKNKSVAAVGKAAGERWKSLSESEKAPYVAKANKLKGEYNKAI
AAYNKGESA