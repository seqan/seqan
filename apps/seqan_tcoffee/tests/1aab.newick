(hmgl_trybr,(hmgt_mouse,(hmgb_chite,hmgl_wheat)))
