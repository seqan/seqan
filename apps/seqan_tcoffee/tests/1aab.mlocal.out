>hmgl_trybr
KKDSNAPKRAMTSFMFFSSDFRS----KHSDLS-IVEMSKAAGAAWKELGPEERKVYEEM
AEKDKERYKREM----------------
>hmgt_mouse
-----KPKRPRSAYNIYVSESFQEAKDDSAQGK-----LKLVNEAWKNLSPEEKQAYIQL
AKDDRIRYDNEMKSWEEQMAE-------
>hmgb_chite
---ADKPKRPLSAYMLWLNSARESIKRENPDFK-VTEVAKKGGELWRGL--KDKSEWEAK
AATAKQNYIRALQEYER----NGG----
>hmgl_wheat
--DPNKPKRAPSAFFVFMGEFREEFKQKNPKNKSVAAVGKAAGERWKSLSESEKAPYVAK
ANKLKGEYNKAIAAYNK-------GESA
